doc/examples/audiopool/lib/main.dart

```diff
@@ -23,14 +23,14 @@ AudioPool pool = AudioPool('laser.mp3');
 class MyGame extends BaseGame with TapDetector {
   static final black = BasicPalette.black.paint;
 
-  MyGame(Size screenSize) {
+  MyGame(Vector2 screenSize) {
     size = screenSize;
   }
 
   @override
   void render(Canvas canvas) {
-    canvas.drawRect(Rect.fromLTWH(0.0, 0.0, size.width, size.height), black);
-    final p = Position.fromSize(size).div(2);
+    canvas.drawRect(Rect.fromLTWH(0.0, 0.0, size.x, size.y), black);
```

Comment:
this reminds of `toZeroRect()`, `toOriginRect` or something from 1.0 :p

maybe it could be a method on the Position class

Reply:
Or just toRect in the this case maybe? A two dimensional vector should be pretty obvious that it should be that `Rect`?